build/screener/config.js

```diff
@@ -0,0 +1,2 @@
+require('ts-node').register()
+module.exports = require('./screener.config').default
```

Comment:
1. I propose to move `tsPaths` there, too:

```diff
+tsPaths.register({
+  baseUrl: config.path_base,
+  paths: compilerOptions.paths,
+})
```

2. I propose to rename this back to `screener.config.js` while, `screener.config.ts` will be `screener.states.ts` and will provide only `states` for config section. What do think?

Reply:
my plan was to get rid of the `.js` file altogether but I think screener API does not accept that.. why not have most of the code in Typescript if it's possible? it's a typescript project after all.

Moving all the code to typescript gave me a better understanding of what's happening there as well since I get instant access to the API through correct typings.

What would be the reasons to go back to JS?

Reply:
resolved per our discussion